projects/simple-if-else-dictionary/app.py

```diff
@@ -0,0 +1,30 @@
+#Simple if else Dictionary.
+import json
```

Comment:
import only required function 
```suggestion
from json import load
```